inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp

```diff
@@ -83,6 +83,18 @@ class LSTMCellStage final : public StageNode {
         bool useTempBuffer = (nCells > 1);
         IE_ASSERT((numTempBuffers() == 1 && useTempBuffer) || !useTempBuffer);
 
+        VPU_THROW_UNLESS(inputEdges().size() == 5,
+                         "input edges: {}", inputEdges().size());
```

Comment:
Please provide more descriptive message. From message "input edges: 3" the intention is unclear. Is it too big, too small or what?

Reply:
Obviously, it must equal to 5....

Or do you mean, that VPU_THROW_UNLESS would not print the assertion expression?

Reply:
I'm not sure about that, anyway it's better not to rely on it, In your PR a little bit lower you provided expected value in the message, though

Reply:
OK, now I see: VPU_THROW_UNLESS does not print the assert expression. 😞 

So I will provide more descriptive message here with my next commit.

Thanks for pointing to this!